# /home/yaoyao/Documents/myProjects/ROS2WithSPCK/src/simpack_control/msg/SimpackY.msg

# 当前仿真时间戳(秒)
float64 sim_time

# 直接写28个字段
float64 Y_Yw01
float64 Y_Yw02
float64 Y_Yw03
float64 Y_Yw04

float64 Y_Yaw01
float64 Y_Yaw02
float64 Y_Yaw03
float64 Y_Yaw04

float64 Y_WL01
float64 Y_WR01
float64 Y_WL02
float64 Y_WR02
float64 Y_WL03
float64 Y_WR03
float64 Y_WL04
float64 Y_WR04

float64 Y_Yt01
float64 Y_Yt02
float64 Y_Yawt01
float64 Y_Yawt02

float64 Y_SpeedDiff_FrontA
float64 Y_SpeedDiff_FrontB
float64 Y_SpeedDiff_FrontC
float64 Y_SpeedDiff_FrontD
float64 Y_SpeedDiff_RearA
float64 Y_SpeedDiff_RearB
float64 Y_SpeedDiff_RearC
float64 Y_SpeedDiff_RearD
